Pipfile

```diff
@@ -7,7 +7,7 @@ name = "pypi"
 
 [packages]
 
-cryptography = "*"
+cryptography = "==2.2"
```

Comment:
Idea:
what if we say `">=2.2"`?

Reply:
Well, we can do `"*"`. I just need to confirm from @KPrasch or @jMyles that this will indeed use the latest version.

Reply:
Technically, >=2.2 is probably the correct one if you use the new API